Hashids/Hashids.swift

```diff
@@ -50,230 +48,205 @@ public typealias Hashids = Hashids_<UInt32>
+            if newSepsLength > sepsLength {
+                let diff = self.alphabet.startIndex.advancedBy(newSepsLength - sepsLength)
+                let range = 0..<diff
+                self.seps += self.alphabet[range]
+                self.alphabet.removeRange(range)
+            } else {
+                let pos = self.seps.startIndex.advancedBy(newSepsLength)
+                self.seps.removeRange(pos+1..<self.seps.count)
             }
         }
         
-        shuffle(&self.alphabet, salt: self.salt);
+        shuffle(&self.alphabet, salt: self.salt)
         
-        let safeGuard = Int(ceil(Double(alphabetLength)/HashidsOptions.GUARD_DIV));
-        if(alphabetLength < 3)
-        {
-            let seps_guard = self.seps.startIndex.advancedBy(safeGuard);
-            let range = 0..<seps_guard;
-            self.guards += self.seps[range];
-            self.seps.removeRange(range);
-        } else
-        {
-            let alphabet_guard = self.alphabet.startIndex.advancedBy(safeGuard);
-            let range = 0..<alphabet_guard;
-            self.guards += self.alphabet[range];
-            self.alphabet.removeRange(range);
+        let safeGuard = Int(ceil(Double(alphabetLength)/HashidsOptions.GUARD_DIV))
+        if alphabetLength < 3 {
+            let seps_guard = self.seps.startIndex.advancedBy(safeGuard)
+            let range = 0..<seps_guard
+            self.guards += self.seps[range]
+            self.seps.removeRange(range)
+        } else {
+            let alphabet_guard = self.alphabet.startIndex.advancedBy(safeGuard)
+            let range = 0..<alphabet_guard
+            self.guards += self.alphabet[range]
+            self.alphabet.removeRange(range)
         }
     }
     
     // MARK: public api
 
-    public func encode(value:Int...) -> String?
-    {
-        return encode(value);
+    public func encode(value: Int...) -> String? {
+        return encode(value)
     }
     
-    public func encode(values:[Int]) -> String?
-    {
-        let ret = _encode(values);
+    public func encode(values: [Int]) -> String? {
+        let ret = _encode(values)
         return ret.reduce(String(), combine: { (var so, i) in
-            let scalar:UInt32 = numericCast(i);
-            so.append(UnicodeScalar(scalar));
+            let scalar:UInt32 = numericCast(i)
+            so.append(UnicodeScalar(scalar))
             return so
-        });
+        })
     }
     
-    public func decode(value:String!) -> [Int]
-    {
-        let trimmed = value.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet());
-        let hash:[Char] = trimmed.unicodeScalars.map{ numericCast($0.value) };
-        return self.decode(hash);
+    public func decode(value: String) -> [Int] {
+        let trimmed = value.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
+        let hash: [Char] = trimmed.unicodeScalars.map{ numericCast($0.value) }
```

Comment:
It's needed, if I remove it get a compiler error, `ambiguous without more context`